src/session/content/message_row/indicators.rs

```diff
@@ -18,6 +18,26 @@ mod imp {
             <property name="spacing">3</property>
           </object>
         </property>
+        <child>
+          <object class="GtkBox" id="reply_count_box">
+            <property name="spacing">3</property>
+            <child>
+              <object class="GtkImage">
+                <property name="icon-name">mail-reply-sender-symbolic</property>
+                <property name="pixel-size">12</property>
```

Comment:
There's a cooler trick to set the same size as the text. You can set the `-gtk-icon-size` css property to the same value as the font size (8pt in this case).

Reply:
Mh, nevermind, this makes the check indicators too small. I think we have to request a custom icon for replies, because this is a bit too big. For now let's use `-gtk-icon-size: 14px`, which seems to work best.